client/app/cloud/project/compute/loadbalancer/cloud-project-compute-loadbalancer.controller.js

```diff
@@ -1,11 +1,14 @@
 class CloudProjectComputeLoadbalancerCtrl {
   constructor(
-    $q, $translate, $state, $stateParams, CloudProjectComputeLoadbalancerService,
+    $q, $translate, $state, $stateParams, atInternet, CloudProjectComputeLoadbalancerService,
     OvhApiCloudProjectIplb, OvhApiIpLoadBalancing, CucCloudMessage, OvhApiMe, URLS,
   ) {
     this.$q = $q;
     this.$translate = $translate;
     this.$state = $state;
+    this.$window = $window;
```

Comment:
`$window` is not injected in parameters https://github.com/ovh-ux/ovh-manager-cloud/pull/1318/files#diff-bf1caf7cd19cbf4ce80cc7d4d3708aeeR3